commons/passive/garmin/garmin_generic_accelerometer.avsc

```diff
@@ -0,0 +1,43 @@
+{
+   "namespace":"org.radarcns.passive.garmin",
```

Comment:
What will be the method of integration? Is this through the pRMT app or a source connector using the REST-APIs extending (the [RADAR-REST-CONNECT](https://github.com/RADAR-base/RADAR-REST-Connector)?) if it is using the source connector, please using the namespace `org.radarcns.connect`.

Reply:
Think the intention here is to use the Companion SDK https://developer.garmin.com/health-sdk/overview/, so via the pRMT app. 

@yatharthranjan presume a separate schema serving the Garmin Connect REST-API https://developer.garmin.com/health-api/overview/ is required for that mode of data access?

Reply:
Hi, yes those will be separate. These are for integration through the pRMT app.

Reply:
I see. :+1:

Reply:
I think the intention is to see if we can get the pRMT integration working (it provides the most data/real-time etc), it's also slightly more challenging to implement. The communication paradigm is interestingly not direct with the device but mediated by the Companion SDK. We can later look at implementing the REST-connector to get the processed data (a la Fitbit).

Reply:
I see. Thanks for the context.